src/components/UI/BuyButton/BuyButton.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
```

Comment:
Move this component under `src/third-party/components` (create those directories if not exists yet)